test/acceptance/tests.py

```diff
@@ -923,6 +963,45 @@ def test_staff_override_at_beginning(self):
         ])
 
 
+class FullWorkflowRequiredTest(FullWorkflowBaseTest):
+    """
+    Tests of complete workflows, combining multiple required steps together.
+    """
+    def setUp(self):
+        super(FullWorkflowRequiredTest, self).setUp(problem_loc_key="full_workflow_required")
+
+    @retry()
+    @attr('acceptance')
+    def test_train_self_peer_staff(self):
+        """
+        Scenario: complete workflow that included staff required step.
+
+        Given that I have created a submission, completed training, and done a self assessment
+        And a second learner has also created a submission, training, and self assessment
+        Then I can assess a learner
+        And when another learner assesses me
+        Then I see my score based on the peer assessment
+        And when a staff member submits a score
+        Then I see the staff score
+        And all fields in the staff area tool are correct
+        """
+        learner = self.do_train_self_peer()
+
+        # Now do a staff assessment.
+        self.do_staff_assessment(options_selected=self.STAFF_OVERRIDE_OPTIONS_SELECTED)
+
+        # At this point, the learner sees the score (1).
+        self.browser.refresh()
+        self._verify_staff_grade_section(self.STAFF_GRADE_EXISTS, None)
+        self.assertEqual(self.STAFF_OVERRIDE_SCORE, self.grade_page.wait_for_page().score)
+
+        # Note that PEER ASSESSMENT isn't shown here - it gets cut off the page in this case
```

Comment:
Nevermind - talked to Christina, realized that having a calculated peer grade is not a prerequisite for having a final grade in this scenario, so it makes sense that there wouldn't be a peer grade section in this view.